### DISCUSSION TITLE
Send JSON in body to REST API

### DISCUSSION THREAD

**Discussion by darkCloud77** [nextgenhealthcare/connect#4949] [Q&A]
I have data coming in HL7 format, converting it to JSON using the outbound JSON message template.  That part appears to work.  I then set a destination to HTTP Sender, set a good URL, and set the Content to Text, UTF-8, value ${message.encodedData}.  

Every single destination execution fails with "ER7Serializer error
ERROR MESSAGE: Error converting XML to ER7
org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog."

It never makes it to the endpoint.

Please please please help.  I've spent many hours Googling and nothing.  Thank you.

**Comment by JonEBee** [comment]
Did you set the Channel Data Type for the Destination Response outbound to JSON, using the Data Types dialog under the Summary tab?  Try setting the Content to "text/plain" or "application/json".
